marketplacetests/tests/test_marketplace_make_an_in-app_payment.py

```diff
@@ -0,0 +1,71 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette import Wait
+
+from marketplacetests.payment.regions.in_app import InAppPayment
+from marketplacetests.payment.app import Payment
+from marketplacetests.marketplace_gaia_test import MarketplaceGaiaTestCase
+from gaiatest.apps.homescreen.app import Homescreen
+from marketplacetests.marketplace.app import Marketplace
+from gaiatest.apps.homescreen.regions.confirm_install import ConfirmInstall
+
+
+class TestMakeInAppPayment(MarketplaceGaiaTestCase):
+
+    APP_NAME = 'Testing In-App-Payments'
+    APP_TITLE = 'In-App-Payments'
+
+    def setUp(self):
+
+        MarketplaceGaiaTestCase.setUp(self)
+        self.connect_to_local_area_network()
+
+        self.homescreen = Homescreen(self.marionette)
+
+        if not self.apps.is_app_installed(self.APP_NAME):
+
+            marketplace = Marketplace(self.marionette, self.MARKETPLACE_DEV_NAME)
+            marketplace.launch()
+
+            results = marketplace.search(self.APP_NAME)
+            results.install_searched_app(self.APP_NAME)
+
+            # Confirm the installation and wait for the app icon to be present
+            confirm_install = ConfirmInstall(self.marionette)
+            confirm_install.tap_confirm()
+
+            self.device.touch_home_button()
+
+        self.apps.switch_to_displayed_app()
+        self.homescreen.wait_for_app_icon_present(self.APP_NAME)
+
+    def test_make_an_in_app_payment(self):
+
+        email = self.testvars['marketplace']['username']
```

Comment:
So, yes payment cannot be done using a new account because region needs to be changed to USA before completing the payment.
This requires more extra steps (creating pin, opening Marketplace, log in there and start change region process) and then switch back to payment frame. @bobsilverberg do you thinks that it's worth adding all these extra steps?
![2015-01-13-19-05-06](https://cloud.githubusercontent.com/assets/2787766/5724917/84ec713a-9b57-11e4-8aac-b576e22a8c3b.png)